packages/asl-pages/pages/project/read/views/components/hba.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Link, Snippet, Inset, Utils } from '@ukhomeoffice/asl-components';
+
+const HBA = ({ version, project, canReplaceHBA }) => {
+  if (!version?.hbaToken) {
+    return null;
+  }
+
+  return (
+    <div>
+      <h3>
+        <Snippet>downloads.hba.heading</Snippet>
+      </h3>
+
+      <div>
+        <p>
+          <a href={`/attachment/${version.hbaToken}`} download={version.hbaFilename}>
+            <Snippet>otherDocuments.links.hba</Snippet>
+          </a>
+        </p>
+        <p>
+          {canReplaceHBA && (
+            <Link
+              page="project.replaceHba"
+              project={project}
+              label={<Snippet>otherDocuments.links.replaceHba</Snippet>}
+            />
+          )}
+        </p>
+      </div>
+
+      {Array.isArray(version.hbaReplaced) && version.hbaReplaced.length > 0 && (
+        <div className="govuk-hint">
+          {(() => {
+            const latest = version.hbaReplaced.at(-1);
+
+            return (
+              <details>
+                <summary>HBA file replaced – view details</summary>
+
+                <Inset>
+                  <div className="govuk-!-margin-bottom-2" style={{ display: 'flex' }}>
+                    <strong style={{ width: '120px' }}>Date</strong>
+                    <span>
+                      {latest.uploadedAt
+                        ? Utils.formatDate(new Date(latest.uploadedAt), Utils.DATE_FORMAT.medium)
```

Comment:
You'll need to add medium to the @asl-components standard date formats - currently only has long and short. https://github.com/UKHomeOffice/aspel-workspace/blob/main/packages/asl-components/src/utils.js#L40-L43. This will technically still work, but as `Utils.DATE_FORMAT.medium` is undefined it will use the default long format e.g. 18 November 2025 instead of 18 Nov 2025.

You could use the pages constant instead https://github.com/UKHomeOffice/aspel-workspace/blob/main/packages/asl-pages/constants/index.js but I think @asl-components formatter is the best one to try and make the standard for ASPeL. I've updated the cat-e one to use the components formatter, which I'll include in the next cat-e ticket's PR.